wgsl/index.bs

```diff
@@ -12188,7 +12236,10 @@ Note: The result is not mathematically meaningful if `e` &le; 0.
     <td>Description
     <td>Returns `e1 * 2`<sup>`e2`</sup>, except:
     * The result may be zero if `e2` + *bias* &le; 0.
-    * The result is any value of type `T` if `e2` &gt; *bias* + 1.
+    * If `e2` &gt; *bias* + 1
```

Comment:
This is a tightening of the spec.  This was the only case where an indeterminate value could be generated as const-expr and override-expr, but it doesn't have to be that way.

That way we can make "indeterminate value" only apply to runtime expressions.